addons/web_editor/static/lib/odoo-editor/test/spec/utils.test.js

```diff
@@ -391,7 +382,7 @@ describe('Utils', () => {
             const [div] = insertTestHtml(
                 '<div><div><p>ab<span>cd</span><b><i><u>ef</u></i></b></p></div></div>',
             );
-            const result = lastLeaf(div, true);
+            const result = lastLeaf(div, isBlock);
```

Comment:
I feel like this strengthens my case for "filter" rather than "filterOut": if I just read this line my first thought is that this gives me the last leaf of the `div` that is a block. In fact it gives me the last sibling of first descendant that is _not_ a block.